src/tools/installEpics.pl

```diff
@@ -57,19 +57,21 @@
     my $temp   = "$install_dir/TEMP.$name.$$";
     my $target = "$install_dir/$name";
 
-    if (-f $target) {
-        next if -M $target < -M $source and -C $target < -C $source;
-        # Remove old target, making sure it is deletable first
-        chmod 0777, $target;
-        unlink $target;
+    if ($source ne $target) {
```

Comment:
I have switched to comparing absolute paths as you suggested. That said, it seemed that I need to directly use `abs_path` from `Cwd` instead of the EPICS one, which (as I understand) is installed by `installEpics.pl`.

I believe that this is fine; the limitations described in `EPICS::Path` about `abs_path` shouldn't apply here as the `source` should already exist, and the target directory is created just above.